### PR TITLE
DO-42: Implemented the launch url for branding.

### PR DESCRIPTION
**Pull request by ayushmalik2110**

### CODE REVIEW

File: src/utils/core/coreUtils.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * @param brandingObj
+ * @returns {null}
+ */
+export const formatBrandingElement = ( brandingObj = {} ) => {
+  const { branding } = brandingObj;
+  if(!isEmpty(branding)) {
+    // The below code segment convert the key like 'primary_color' to css variable i.e. '--primary-color'.
+    let formattedBrandingElement = {};
+    Object.keys(branding).forEach(key => {
+      debugger;
```
Comment (mohitsaladiya) [inline src/utils/core/coreUtils.js:19]: debugger?

File: src/assets/css/common/appHeader/_AppHeader.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@
   position: fixed;
   border-bottom: 1px solid black;
   top: 0;
-  background-color: var(--backgroundColorPrimary);
```
Comment (mohitsaladiya) [inline src/assets/css/common/appHeader/_AppHeader.scss:9]: let it be variables for now

File: config/tenant/default/app.json [hunk 3]
```diff
@@ -21,5 +21,21 @@
       "config": "profileView",
       "isActive": true
     }
-  ]
+  ],
```
Comment (mohitsaladiya) [inline config/tenant/default/app.json:24]: move this to response formatter

File: src/constants/core/coreConstants.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ */
+
+/**
+ * These are the css variable name of branding element.
+ * @type {{tertiary_color: string, copyright_url: string, terms_condition_url: string, logo_ulr: string,
+ * secondary_color: string, primary_color: string, primary_font: string,
+ * secondary_font: string, tertiary_font: string}}
+ */
+export const BRANDING_ELEMENT = {
+  "logo_ulr": "--logo-url",
```
Comment (mohitsaladiya) [inline src/constants/core/coreConstants.js:12]: logo_ulr ??

File: config/tenant/default/app.json (hunk 3 above)
Comment (ayushmalik2110) [reply config/tenant/default/app.json:24]: done.

Comment (ayushmalik2110) [reply config/tenant/default/app.json:24]: done.

File: src/assets/css/common/appHeader/_AppHeader.scss (hunk 2 above)
Comment (ayushmalik2110) [reply src/assets/css/common/appHeader/_AppHeader.scss:9]: done.

File: src/utils/core/coreUtils.js (hunk 1 above)
Comment (ayushmalik2110) [reply src/utils/core/coreUtils.js]: Removed.

**Pull request merged**